.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,26 @@
+name: Commitlint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+jobs:
+  lint:
+    runs-on: ubuntu-18.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v2.0.3
+        with:
+          configFile: './.commitlintrc.json'
+          helpURL: https://rook.io/docs/rook/latest/development-flow.html#commit-structure
```

Comment:
> Why url into rook's repository? and in any case, it is a dead link.

this was because the file was originally copied from rook. I removed it already yesterday when I noticed ...

Reply:
fwiw, while the link is in fact dead, here is the doc: https://github.com/rook/rook/blob/master/Documentation/Contributing/development-flow.md

and it does have a section "commit structure" ...